src/Core/src/Handlers/DatePicker/DatePickerHandler.Android.cs

```diff
@@ -59,6 +62,10 @@ protected override void DisconnectHandler(MauiDatePicker platformView)
 
 			platformView.ViewAttachedToWindow -= OnViewAttachedToWindow;
 			platformView.ViewDetachedFromWindow -= OnViewDetachedFromWindow;
+
+			if (platformView.Context is MauiAppCompatActivity context)
```

Comment:
I don't think we can depend on the activity being of type `MauiAppCompatActivity`

Embedding scenarios aren't always going to inherit from `MauiAppCompatActivity`